src/components/pod/__spec__.js

```diff
@@ -373,6 +373,12 @@ describe('Pod', () => {
       expect(div.attributes['data-foo'].value).toEqual("bar");
     });
 
+    it('does not apply title prop to containing elements', () => {
+      const wrapper = shallow(<Pod title="some-title" />);
+      const re = /title=\"some-title\"/;
+      expect(wrapper.html().match(re)).toBeNull();
```

Comment:
I think you could use [Enzyme's is](https://airbnb.io/enzyme/docs/api/ShallowWrapper/is.html) here instead of regex e.g. `expect(wrapper.is('[title]')).toBe(false)`.